packages/core/src/bus/Exceptions/IntegrationEventHandlerNotFound.ts

```diff
@@ -0,0 +1,7 @@
+export class IntegrationEventHandlerNotFound extends Error {
+  constructor(...eventNames: string[]) {
+    super(
+      `For events ${eventNames.join(', ')} was not registered any handlers.`
+    );
+  }
+}
```

Comment:
First of all, you need to override `name`. Secondly, using es5, `tsc` will break a prototype chain. You have to set the prototype explicitly.

```suggestion
export class IntegrationEventHandlerNotFound extends Error {
  constructor(...eventNames: string[]) {
    super(
      `For events ${eventNames.join(', ')} was not registered any handlers.`
    );
    this.name = new.target.name;
    Object.setPrototypeOf(this, new.target.prototype);
  }
}
```